src/patterns/components/offline-notification/overview.hbs

```diff
@@ -0,0 +1,7 @@
+---
+order: 10
+notes: |
+  In page notification about lack of internet connectivity.
+---
+
+{{{embed "patterns.components.offline-notification.example"}}}
```

Comment:
I just copied how pagination is set up. I'm guessing it is done this way so the example can be embedded in other pages, which we'll probably want to do for this one eventually.